src/share/poudriere/image_firmware.sh

```diff
@@ -134,7 +134,7 @@ firmware_generate()
 	espfilename=$(mktemp /tmp/efiboot.XXXXXX)
 	make_esp_file ${espfilename} ${ESP_SIZE} ${WRKDIR}/world/boot/loader.efi
 	mkimg -s gpt -C ${IMAGESIZE} -b ${mnt}/boot/pmbr \
-		-p efi:=${espfilename} \
+		-p efi/efiboot:=${espfilename} \
```

Comment:
I have been using `efiboot0` to keep it just like the output of bsdinstall (zfsboot), confirmed with: `gpart show -l`.
Although, the label appears as `efiesp` for [vimages](https://github.com/freebsd/freebsd-src/blob/d3b6d70e0dfaa4e9496bddcbc212d7d3ea392800/release/tools/vmimage.subr#L304), there is no standard yet, so this comment can be safely ignored.

Reply:
EDIT: I have submitted a small rant on the topic: https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=278480 .

Reply:
I think we should standardize with what the installer is doing: `efiboot0` It's likely the most widely deployed...